cfn-resources/federated-settings-identity-provider/mongodb-atlas-federatedsettingsidentityprovider.json

```diff
@@ -90,7 +75,13 @@
   },
   "readOnlyProperties": [
     "/properties/FederationSettingsId",
-    "/properties/OktaIdpId"
+    "/properties/OktaIdpId",
+    "/properties/Profile"
```

Comment:
The profile should be a create-only properties as it can be specified by the user only during resource creation.